app/controllers/application_controller.rb

```diff
@@ -15,6 +15,12 @@ def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:account_update, keys: %i[first_name last_name weight height belt years_of_experience address description gender photo training_style])
   end
 
+  def set_notifications
```

Comment:
This is for universal access of notifications so I can add purple circles on notification in footer.